pkg/cloud/vsphere/context/cluster_context.go

```diff
@@ -175,6 +177,20 @@ func (c *ClusterContext) Pass() string {
 	return c.pass
 }
 
+// Server returns the vCenter or ESXi server address
+func (c *ClusterContext) Server() string {
+	return c.server
+}
+
+func (c *ClusterContext) GetSSH() *SSH {
```

Comment:
Hi @moshloop,

I would like SSH to be its own package. Please refer to https://github.com/akutz/simple-k8s-test-env/tree/feature/sk8-app/pkg/net/ssh for an example of an SSH package that supports direct and bastion access.

Reply:
I moved ssh into its own package, I don't think we *need* bastion access for ESXi, and the sk8 ssh package does not support username/password auth, it also has a few other deps that don't make it a simple copy and paste so I think it would better to leave this implementation as is, and bring on other features from sk8 as and when they are needed.